File: jan23/jan23.java
public class FrameCounter {
    public static int frames(int minutes, int fps) {
        int seconds = minutes * 60;
        int totalFrames = seconds * fps;
        return totalFrames;
    }

    public static void main(String[] args) {
        // Examples
        System.out.println(frames(1, 1));    // Output: 60
        System.out.println(frames(10, 1));   // Output: 600
        System.out.println(frames(10, 25));  // Output: 15000
    }
}

File: jan23/jan23.js
function frames(minutes, fps) {
    const seconds = minutes * 60;
    const totalFrames = seconds * fps;
    return totalFrames;
}

// Examples
console.log(frames(1, 1));    // Output: 60
console.log(frames(10, 1));   // Output: 600
console.log(frames(10, 25));  // Output: 15000